brackend/db/models.py

```diff
@@ -72,8 +85,8 @@ class Bracket(Base):
     tournament = Column(Integer, ForeignKey("tournaments.id"))
     rounds = relationship("Round", backref="brackets")
 
-    def to_json(self):
-        return {"id": self.id, "name": self.name, "tournament": self.tournament}
+    def to_json(self, participants=[]):
```

Comment:
Don't add mutable default args in python. If you append to this list, your next function call will have the appended elements.

Better is
 
```suggestion
    def to_json(self, participants=None):
        if participants is None:
            participants = []
```

Reply:
Although tbh, if we don't feeed in participants I'd rather it be None, signifying we didnt feed in participants and that list COULD be non empty, than the empty list, which signifies we did feed in all the participants but there just werent any.

Reply:
Oh shit I did not know this about the mutable args, lol even though it's like #1 on a list of common python "gotchyas"

![image](https://user-images.githubusercontent.com/11068205/149963985-30b09b56-f108-4ce9-b4ae-98c0689d1d68.png)